families/track_and_trade/server/api/users.js

```diff
@@ -27,9 +27,27 @@ const create = user => {
       return db.insert(_.assign({}, user, {password: hashed}))
         .catch(err => { throw new BadRequest(err.message) })
     })
-    .then(() => _.omit(user, 'password'))
+    .then(() => auth.createToken(user.publicKey))
+    .then(token => ({
+      authorization: token,
+      signing: user.privateKey
```

Comment:
Is this the encrypted private key?

Reply:
Yes. The encryption is 100% client-side though. You could pass the server a plain text private key, and it wouldn't know the difference. It just takes whatever you give it, and stores it.

In this case, the user just sent in the private key, so sending it back is probably redundant, but it keeps it consistent with the response from the auth route.

Reply:
Would suggest naming the field `encryptedSigningKey`, `signing` doesn't really give much information about what the field is, then.

Reply:
Perhaps the same should be said for all the `privateKey` fields.  Making the name of the field `encryptedPrivateKey` will at least give a hint to the user not to send the plain-text private key over the wire.

Reply:
This is a good point. I will do some more thinking about what those names should be.